packages/types/tsconfig.json

```diff
@@ -4,6 +4,6 @@
   "include": ["src/**/*"],
 
   "compilerOptions": {
-    // package-specific options
+    "lib": ["ES6"]
```

Comment:
Note: This setting actually leads to the old code failing at build time, so this should also prevent this happening again for types, at least.